Editor/Resources/EditorWindow/Pages/LandingPage.uss

```diff
@@ -0,0 +1,69 @@
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+@import "./common.uss";
+
+:root {
+    --color-card-border: #0d0d0d;
+    --color-card-background: #2a2a2a;
+}
+
+.cards {
+    flex-direction: row;
+    margin-right: var(--spacing-large-negative);
+}
+
+.card {
+    padding: var(--spacing-small);
+    border-width: var(--border-size-medium);
+    border-color: var(--color-card-border);
+    background-color: var(--color-card-background);
+    flex: 1 0 0;
+    margin-right: var(--spacing-large);
+}
+
+.card__container {
+    flex-direction: row;
+}
+
+.card__text {
+    font-size: var(--font-size-default);
+    -unity-text-align: middle-left;
+    flex: 1 1 auto;
+}
+
+.sample__header {
+    font-size: var(--font-size-extralarge);
+    margin-bottom: var(--spacing-large);
+}
+
+.sample__content {
+    flex-direction: row;
+}
+
+.sample__text {
+    align-items: flex-start;
+}
+
+.sample__image {
+    width: 120px;
+    height: 120px;
+    flex: 0 0 auto;
+    margin-right: var(--spacing-large);
+}
+
+.sample__button {
+    margin-top: 8px;
+}
+
+Image#LandingPageNoAccountImage {
+    --unity-image: resource("Images/Icons/NoAccountIcon");
+}
+
+Image#LandingPageAccountImage {
+    --unity-image: resource("Images/Icons/AccountIcon");
+}
+
+Image#LandingPageSampleImage {
+    --unity-image: resource("Images/LandingPage/SampleGame");
```

Comment:
I found out that this isn't the final image for the SampleGame. It's expected that y'all create a new image for the plugin based on the finalized sample game. Since the sample game stuff is in PR right now, I've added this to the bug tracker and won't block on it